AppController/djinn.rb

```diff
@@ -3476,7 +3487,7 @@ def start_api_services()
       UserAppClient::SSL_SERVER_PORT, USE_SSL)
     uac = UserAppClient.new(@my_private_ip, @@secret)
     begin
-      app_list = uac.get_all_apps()
+      uac.does_app_exist?("not-there")
```

Comment:
@cdonati  there is already a comment above telling that the call is to check the UAServer is responsive. Should I move the comment closer to the call?